lighthouse-core/lib/asset-saver.js

```diff
@@ -102,8 +101,8 @@ async function saveArtifacts(artifacts, basePath) {
   const status = {msg: 'Saving artifacts', id: 'lh:assetSaver:saveArtifacts'};
   log.time(status);
   fs.mkdirSync(basePath, {recursive: true});
-  rimraf.sync(`${basePath}/*${traceSuffix}`);
-  rimraf.sync(`${basePath}/${artifactsFilename}`);
+  fs.rmdirSync(`${basePath}/*${traceSuffix}`, {recursive: true});
```

Comment:
glob works?

Reply:
> glob works?

no, and in fact doing this with `fs.rmdir` also makes no sense. Thanks for catching :)

Reply:
> in fact doing this with `fs.rmdir` also makes no sense

[`fs.rm()`](https://nodejs.org/api/fs.html#fs_fs_rm_path_options_callback) will let us handle files and directories (with and without `recursive: true`, respectively), but not until node 14, unfortunately

Reply:
fixed and added a test to catch this next time